docs/changelog.md

```diff
@@ -178,10 +209,10 @@ compile "org.janusgraph:janusgraph-core:1.0.1"
 * Apache HBase 2.5.8
 * Oracle BerkeleyJE 7.5.11
 * ScyllaDB 5.1.4
```

Comment:
**❓ question:** ‏Should it be 6.2.0? (based on [this line](https://github.com/porunov/janusgraph/blob/release/v1.1.0/pom.xml#L107))

Reply:
That is for Version 1.0.1 and not 1.1.0. We didn't merge it to `v1.0` branch.